src/questions/derivatives.yaml

```diff
@@ -0,0 +1,951 @@
+  answerKeys:
+    - A
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Put option
+      key: A
+    - content: Call Option
+      key: B
+    - content: Swap
+      key: C
+    - content: None of the above
+      key: D
+- uuid: d8f4d21e-5f1c-4ec2-b503-6f78c6470bf6
+  type: SingleChoice
+  content: The amount paid for an option is the
+  hint: NoHint
+  explanation: Premium is the amount that must be paid for an option.
+  answerKeys:
+    - B
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Strike price
+      key: A
+    - content: Premium
+      key: B
+    - content: Yield
+      key: C
+    - content: Commission
+      key: D
+- uuid: 663713ba-03af-4cf1-904b-7e3790289740
+  type: SingleChoice
+  content: An option that can be executed any time upto maturity is called
+  hint: NoHint
+  explanation: American Option can be executed any time upto maturity.
+  answerKeys:
+    - D
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Swap
+      key: A
+    - content: Stock Option
+      key: B
+    - content: European Option
+      key: C
+    - content: American Option
+      key: D
+- uuid: c1ca8227-67e1-48b8-8a60-1cf70ad93a31
+  type: SingleChoice
+  content: Individual stock options are known as-
+  hint: NoHint
+  explanation: Options on individual stocks are known as stock options.
+  answerKeys:
+    - A
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Stock options
+      key: A
+    - content: Futures options
+      key: B
+    - content: American options
+      key: C
+    - content: Individual options
+      key: D
+- uuid: 8bf31665-8afd-46de-af73-3d4755f90069
+  type: SingleChoice
+  content: Options that can only be executed on maturity is called-
+  hint: NoHint
+  explanation: European Option can be executed only at maturity.
+  answerKeys:
+    - C
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Swap
+      key: A
+    - content: Stock Option
+      key: B
+    - content: European Option
+      key: C
+    - content: American Option
+      key: D
+- uuid: 8bf43610-6267-4b29-894e-12b392c60ee6
+  type: SingleChoice
+  content: Options on futures contracts are known as-
+  hint: NoHint
+  explanation: Options on future contracts are referred as futures options.
+  answerKeys:
+    - B
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Swap
+      key: A
+    - content: Futures Options
+      key: B
+    - content: Stock Options
+      key: C
+    - content: Individual Options
+      key: D
+- uuid: 92fb85a6-f95d-4c30-ad38-01ca1bf64287
+  type: SingleChoice
+  content: Which of the following options gives the owner the right to buy a financial instrument at executed price within a specific duration of time?
```

Comment:
Please confirm this. (For both Robin and Lakshya)

Reply:
Robin, I have confirmed this is correct. I can explain in the next meet